DMP-Capstone/TableMaster
# Margaret branch

Process for requesting a seat is complete. Still need to figure out why some messages are not showing but data is being saved to tables and messages are being "sent"

To Test:
AFter logging into account and requesting a seat
Log into the host account and approve request.
**you should see an update to the approved column for request and a message being created with subject "Seat Request Decision for <Whatever title you gave menu>. ***

However the actual message is not being displayed on the view so will try to crack that code tomorrow or later today to be technical.
